### PR TITLE
Enable glibc-compatibility on arm

### LINKED ISSUE #6571: Enable glibc-compatibility on arm

### PR DESCRIPTION
**Pull request by xzhangxian1008**
### What problem does this PR solve?

Issue Number: close #6571

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Comment by solotzg** [comment]
Can you help to investigate whether we can remove https://github.com/pingcap/tiflash/blob/eaf1a4cf886d1d5a86d04ef14eff956778ab4911/libs/libglibc-compatibility/glibc-compatibility.c#L27-L30

**Comment by xzhangxian1008** [comment]
> Can you help to investigate whether we can remove this func safely ?
> 
> https://github.com/pingcap/tiflash/blob/eaf1a4cf886d1d5a86d04ef14eff956778ab4911/libs/libglibc-compatibility/glibc-compatibility.c#L27-L30

okk, I will

**Comment by xzhangxian1008** [comment]
> 

I think we can not remove it before we manually designate the signal stack. However, the returned value could be changed to 16KiB, so that we can save many VSZ.

**Comment by xzhangxian1008** [comment]
> Can you help to investigate whether we can remove this func safely ?
> 
> https://github.com/pingcap/tiflash/blob/eaf1a4cf886d1d5a86d04ef14eff956778ab4911/libs/libglibc-compatibility/glibc-compatibility.c#L27-L30

I think we can not remove it before we manually designate the signal stack. However, the returned value could be changed to 16KiB, so that we can save many VSZ. Morever, though we can use SIGSTKSZ to define the size of signal stack, it's too small to collect enough information.

File: libs/libglibc-compatibility/glibc-compatibility.c [hunk 1]
```diff
@@ -26,7 +26,7 @@ extern "C" {
 
 size_t __pthread_get_minstack(const pthread_attr_t * attr)
 {
-    return 1048576;        /// This is a guess. Don't sure it is correct.
+    return 16384;        /// This is a guess. Don't sure it is correct.
```
Comment (solotzg) [inline libs/libglibc-compatibility/glibc-compatibility.c:29]: Please update comment and add explanation.

**Review by solotzg — APPROVED** [review]

**Comment by solotzg** [comment]
lgtm

**Review by windtalker — APPROVED** [review]
LGTM

File: libs/libglibc-compatibility/glibc-compatibility.c (hunk 1 above)
Comment (xzhangxian1008) [reply libs/libglibc-compatibility/glibc-compatibility.c]: > Please update comment and add explanation.

done

**Review by ywqzzy — APPROVED** [review]

**Review by windtalker — APPROVED** [review]
LGTM

**Pull request merged**